### PR TITLE
SF-2006 Checking overview page to Material

### PR DESCRIPTION
**Pull request by josephmyers**
- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)

### CODE REVIEW

File: src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.scss [hunk 1]
```diff
@@ -13,6 +13,12 @@
 }
 
 #add-question-button {
+  color: white;
```
Comment (Nateowami) [inline src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.scss:16]: Set `color=primary` on the button, and none of this change to CSS rules should be needed.

**Comment by josephmyers** [comment]
[Community Checking Performance.xlsx](https://github.com/sillsdev/web-xforge/files/11473702/Community.Checking.Performance.xlsx)

Above are my very simple test findings

**Review by josephmyers — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)* status: 0 of 5 files reviewed, 1 unresolved discussion (waiting on @Nateowami)

___
*[`src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.scss` line 16 at r1](https://reviewable.io/reviews/sillsdev/web-xforge/1836#gh-1189934907:-NVhe0yjAeuvtyt4cwzV:b-896fix) ([raw file](https://github.com/sillsdev/web-xforge/blob/64d938b6edd74369b4d5f31f0de893709f45820e/src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.scss#L16)):*
<details><summary><i>Previously, Nateowami (Nathaniel Paulus) wrote…</i></summary><blockquote>

Set `color=primary` on the button, and none of this change to CSS rules should be needed.
</blockquote></details>

Done.

**Review by Nateowami — COMMENTED** [review]
> Reviewed 2 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)* status: 2 of 5 files reviewed, all discussions resolved

**Review by Nateowami — COMMENTED** [review]
> Reviewed 2 of 5 files at r2, 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)* status: 4 of 5 files reviewed, all discussions resolved

**Review by Nateowami — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836#-:-NWJbXwb0-UwGx5gB9wK:bnfp4nl)* status: 4 of 5 files reviewed, all discussions resolved

**Review by Nateowami — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)* status: 4 of 5 files reviewed, 1 unresolved discussion (waiting on @josephmyers)

___
*[`src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.spec.ts` line 1021 at r3](https://reviewable.io/reviews/sillsdev/web-xforge/1836#-NWK3c124SPecoZYqIM-:-NWK3c124SPecoZYqIM0:bnjav8b) ([raw file](https://github.com/sillsdev/web-xforge/blob/d4984d3508aad15f8a156be8140c32941a321178/src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.spec.ts#L1021)):*
> ```XML
>   }
> 
>   clickExpanderAtRow(rowIndex: number, fromArchives?: boolean): void {
> ```

This really should be two methods. It would make the tests a lot more understandable. When you see a line like `env.clickExpanderAtRow(0, true);`, there's no way to know that `true` means "fromArchives".

Non-blocking comment though; don't want to hold up the merge of this change.

**Review by Nateowami — APPROVED** [review]
> Reviewed 1 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1836)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @josephmyers)

**Pull request merged**